File: ExcAlgoritmoProgramacao_JS/Lista_Bee/2163_basico.js
import { question } from "readline-sync";

function main() {
    let matriz = [];
    let [n, m] = get_Text().split(` `).map(Number);
    let linha = 0;
    let coluna = 0;

    for (let i = 0; i < n; i++) {
        let numeros = get_Text().split(` `).map(Number);
        matriz.push(numeros);
    }

    for (let i = 0; i < n; i++) {
        for (let j = 0; j < m; j++) {
            if (matriz[i][j] === 42) {
                if (
                    i > 0 && i < n - 1 &&
                    j > 0 && j < m - 1 &&
                    matriz[i - 1][j - 1] === 7 &&
                    matriz[i - 1][j] === 7 &&
                    matriz[i - 1][j + 1] === 7 &&
                    matriz[i][j - 1] === 7 &&
                    matriz[i][j + 1] === 7 &&
                    matriz[i + 1][j - 1] === 7 &&
                    matriz[i + 1][j] === 7 &&
                    matriz[i + 1][j + 1] === 7
                ) {
                    linha = i + 1;
                    coluna = j + 1;
                    break;
                }
            }
        }
        if (linha !== 0 && coluna !== 0) break;
    }

    print(`${linha} ${coluna}`);

}

function get_Text() {
    return question();
}

function print(texto) {
    console.log(texto);
}

main();

File: ExcAlgoritmoProgramacao_JS/Lista_Bee/2163_basico.py
def main():
    matriz = []
    n, m = map(int, get_Text().split(' '))
    linha = 0
    coluna = 0

    for _ in range(n):
        numeros = list(map(int, get_Text().split(' ')))
        matriz.append(numeros)

    for i in range(n):
        for j in range(m):
            if matriz[i][j] == 42:
                if (
                    i > 0 and i < n - 1 and
                    j > 0 and j < m - 1 and
                    matriz[i - 1][j - 1] == 7 and
                    matriz[i - 1][j] == 7 and
                    matriz[i - 1][j + 1] == 7 and
                    matriz[i][j - 1] == 7 and
                    matriz[i][j + 1] == 7 and
                    matriz[i + 1][j - 1] == 7 and
                    matriz[i + 1][j] == 7 and
                    matriz[i + 1][j + 1] == 7
                ):
                    linha = i + 1
                    coluna = j + 1
                    break
                
        if (linha != 0 and coluna != 0): 
            break

    print(f'{linha} {coluna}')
    
def get_Text():
    return input().strip()

main()